### ISSUE TITLE
Make Robohive useable in Kitchen Environment

### ISSUE THREAD

**Issue by Toradus** [intuitive-robots/robohive#3]
The current starting position of the default simulation is located inside the Kitchen (nearly the same cartesian coords as [the robot resting pose](https://github.com/intuitive-robots/irl_polymetis/issues/5) ).
We need to either create a new clean simulation (e.g. copy [the one from SimulationFramework](https://github.com/ALRhub/SimulationFramework/blob/master/models/mj/robot/panda.xml), where the starting position is located somewhere else, or implement a simulation of the Kitchen environment. The XML + Assets for the kitchen env are already created (Max / Marcel have them), feel free to ask for a copy if you want to implement it.

**Comment by omeryagmurlu** [comment]
If we start polymetis with the correct configuration (eg resting pose set correctly: https://github.com/intuitive-robots/irl_polymetis/issues/5#issuecomment-2009462978), then this is a non-issue since robohive uses the resting pose from polymetis by default:

https://github.com/intuitive-robots/robohive/blob/719a194a50fe79aeb4fd51a9cd5daeba2a83c752/robohive/robot/hardware_franka.py#L155

**Comment by Toradus** [comment]
Hmm, we need to recheck this!
From my memory the simulation starts in a slightly different location then the resting pose and the robot always made a small "jump" from resting_pose -> sim_start_pose
Maybe the simulation is just configurated wrong and provides a "reset_pos" to the reset command you linked.

**Comment by Toradus** [comment]
Yeah, we simply have to adapt [this line](https://github.com/mbreuss/irl_robohive/blob/ba989ed86b5576a3649685d63010f6de2df24969/data_collection/env_teleop_wrapper.py#L46C18-L46C23)
good catch!
